test/components/urlBarSuggestionsTest.js

```diff
@@ -150,11 +151,16 @@ describe('urlBarSuggestions', function () {
 
   it('selection is not reset when pressing non-input key', function * () {
```

Comment:
Ok this is failing, because your history would be different. I reset now history before the test